scripts/migrations/002-remove-webview-flags.test.js

```diff
@@ -131,29 +131,25 @@ const tests = [
   },
 ];
 
-const testFixWebViewFlags = (logger = console) => {
-  let hasErrors = false;
-  for (let i = 0; i < tests.length; i++) {
-    let expected = JSON.stringify(tests[i]['output'], null, 2);
-    let output = JSON.stringify(
-      JSON.parse(JSON.stringify(tests[i]['input']), removeWebViewFlags),
-      null,
-      2,
-    );
+describe('migration scripts', () => {
+  it('`removeWebViewFlags()` works correctly', () => {
+    let hasErrors = false;
+    for (let i = 0; i < tests.length; i++) {
+      let expected = JSON.stringify(tests[i]['output'], null, 2);
+      let output = JSON.stringify(
+        JSON.parse(JSON.stringify(tests[i]['input']), removeWebViewFlags),
+        null,
+        2,
+      );
 
-    if (output !== expected) {
-      logger.error(chalk`{red WebView flags aren't removed properly!}
+      if (output !== expected) {
```

Comment:
Could we opt for using `assert.deepStrictEqual(output, expected)` here?

Reply:
If we elect to use `assert.deepStrictEqual(output, expected)` we can probably remove the whole JSON normalization (parsing and then stringification of data). Should I do that?

Reply:
Yeah, that's actually a great point, we could simply write this as:

```js
for (const test of tests) {
  assert.deepStrictEqual(output, expected);
}
```

Reply:
I used the following:
```
    for (const test of tests) {
      const expected = test.output;
      const output = JSON.parse(JSON.stringify(test.input), removeWebViewFlags);
      assert.deepStrictEqual(output, expected);
    }
```